libs/coin-modules/coin-ton/src/bridge/bridgeHelpers/txn.ts

```diff
@@ -33,6 +40,32 @@ export async function getTransactions(
   return txs;
 }
 
+export async function getJettonTransfers(
+  addr: string,
+  startLt?: string,
+): Promise<TonJettonTransfer[]> {
+  const txs = await fetchJettonTransactions(addr, { startLt });
+  if (txs.length === 0) return txs;
+  let tmpTxs: TonJettonTransfer[];
+  let isUncompletedResult = true;
+
+  while (isUncompletedResult) {
```

Comment:
I think you can check whether the txs.length<256, if so, you can return txs directly. 
because in most case, the number of transactions is between. you can avoid calling "fetchJettonTransactions" twice.
if you check txs.length<256 in the loop, I think you can simplify your code.

Reply:
But what happen if the address has more than 256? We are requesting all of the token transfers (related with the different tokens)